gap/matrix.gi

```diff
@@ -763,6 +767,11 @@ InstallGlobalFunction( FindKernelLowerLeftPGroup,
     return true;
   end );
 
+# computes a straight line program (SLP) of <g> for a p-group in <ri> (that
+# correspond to a lower left triangular matrix;
```

Comment:
```suggestion
# corresponds to a lower triangular matrix group;
```
or: "... that consists of lower triangular matrices", or  "... a matrix group consisting of lower triangular matrices" or so